Quick_Start.ipynb

```diff
@@ -301,134 +301,203 @@
    "id": "5bacb2da",
    "metadata": {},
    "source": [
-    "This tells us that this project had only a self-consistent field simulation (`scf`).\n",
-    "\n",
-    "We can take a look some information about this calculation by running:\n"
```

Comment:
Is this line not helpful?
`"We can take a look some information about this calculation by running:`